modern/src/DevicesList.js

```diff
@@ -145,7 +151,9 @@ const DeviceView = ({ updateTimestamp, onMenuClick }) => {
             height={height}
             itemCount={items.length}
             itemData={{ items, onMenuClick }}
-            itemSize={72 + 1}
+            itemSize={72}
+            overscanCount={10}
+            innerRef={listInnerEl}
```

Comment:
Why do we need this parameter?

Reply:
@tananaev It's to allow styling of the list container. We need space at the bottom and top of the list. Putting the space anywhere else result in the list item being cut. This is the problem you pointed out in point 1 of https://github.com/traccar/traccar-web/issues/878